outlines/1.3-migration

```diff
@@ -0,0 +1,31 @@
+1. Breaking changes (as we'd normally do this)
+
+2. Recommended changes : Modules [link to app structure guide]
+  1. Applications: Move all files to imports, initialize in one spot, link via `import`
+  2. Import *all* symbols from packages, including meteor core packages.
```

Comment:
Well all of this would be just adding additional metadata to the API methods, which is something the docs app is actually very well-equipped to do. For example, maybe we could use the jsdoc `@module` annotation: http://usejsdoc.org/tags-module.html